### PR TITLE
Add docs on how to access users' home directories

### PR DESCRIPTION
**Pull request by yuvipanda**
Fixes https://github.com/2i2c-org/infrastructure/issues/1691

### CODE REVIEW

File: docs/howto/support/home-dir.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+```bash
+python3 deployer exec-homes-shell <cluster-name> <hub-name>
+```
+
+Will open a bash shell with all the home directories of all the users of `<hub-name>`
+in `<cluster-name>` mounted in read-write fashion under `/home/`.
+
+```{warning}
+BE CAREFUL! DO NOT DELETE THINGS HERE WITHOUT BEING SURE YOU WANT THEM GONE.
```
Comment (sgibson91) [inline docs/howto/support/home-dir.md:19]:
```suggestion
BE CAREFUL! DO NOT DELETE THINGS HERE WITHOUT BEING SURE YOU WANT THEM GONE.

If in doubt, **rename files** which will allow the server to start up again and preserve the file for the user so they can correct the file contents.
```

**Review by sgibson91 — APPROVED** [review]
Small suggestion, but LGTM!

**Review by GeorgianaElena — APPROVED** [review]
🚀

**Comment by yuvipanda** [comment]
Thanks @sgibson91 and @GeorgianaElena

**Pull request merged**